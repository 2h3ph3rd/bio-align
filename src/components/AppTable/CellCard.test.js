import React from 'react'
import { render, screen } from '@testing-library/react'
import renderer from 'react-test-renderer'

import CellCard from './CellCard'

const backgroundColor = '#AAAAAA'
const children = <div>TEST</div>

it('renders correctly', () => {
  const tree = renderer
    .create(
      <CellCard
        backgroundColor={backgroundColor}
        children={children}
      ></CellCard>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})

test('children rendering', () => {
  render(
    <CellCard backgroundColor={backgroundColor} children={children}></CellCard>
  )

  const div = screen.getByText('TEST')
  expect(div).toBeInTheDocument()
})
