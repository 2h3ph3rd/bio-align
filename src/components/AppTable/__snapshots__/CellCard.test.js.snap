// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-ch5xyf-MuiPaper-root"
>
  <div>
    TEST
  </div>
</div>
`;
